query-planner-js/src/buildPlan.ts

```diff
@@ -1366,7 +1367,10 @@ class FetchDependencyGraph {
     return rootNodes;
   }
 
-  dumpOnConsole() {
+  dumpOnConsole(msg?: string) {
```

Comment:
Nit: that method isn't used, but I keep it around as it helps when debugging. Adding the message here just save some `console.log` when doing so.